inventory/any.cf

```diff
@@ -603,6 +603,15 @@ bundle common cfe_autorun_inventory_aws
         scope => "namespace",
         if => isvariable("cfe_autorun_inventory_dmidecode.dmi[bios-vendor]");
 
+@if minimum_version(3.22.0)
+      "sys_hypervisor_uuid_readable" -> { "ENT-9931" }
+        expression => isreadable("/sys/hypervisor/uuid", 1);
+@else
```

Comment:
Yay else

Reply:
Yep, I considered adding a comment like "this could be dangerous, with `cat` waiting forever, but not likely and certainly not by experimentation, either the file is there or not, and readable, or mysteriously: not."